connector/connect/server/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -1385,6 +1385,21 @@ class SparkConnectPlanner(val session: SparkSession) {
       accumulator = null)
   }
 
+  private def transformPythonForeachFunction(fun: proto.Foreach): SimplePythonFunction = {
```

Comment:
Could use `PythonUDF` in the request proto and call `transformPythonFunction()` here (actually we many not need this new function).